docs/user/howto/shared-deps.md

```diff
@@ -0,0 +1,165 @@
+# Working with shared dependencies
+
+Hypper charts understand the concept of shared dependency charts.
+
+A chart declared as a shared dependency is a chart that more than one chart may
+depend on. Charts deployed as shared dependencies are the analogous of system
```

Comment:
```suggestion
depend on, once deployed to a cluster, the deployment may be reused by multiple other deployments. Charts deployed as shared dependencies are the analogous of system
```

Reply:
fixed in https://github.com/rancher-sandbox/hypper/pull/131/commits/7d9f6332ca1011b74596d6be572a7a0c5c84da5d.